index.html

```diff
@@ -430,6 +440,15 @@ <h2><dfn>disconnect()</dfn> method</h2>
         <a>relevant global object</a>, and also empty <a>context object</a>'s
         <a>observer buffer</a>.</p>
       </section>
+      <section>
+        <h2><dfn>supportedEntryTypes</dfn> attribute</h2>
+        <p>The <a>supportedEntryTypes</a>'s attribute getter must return a list
```

Comment:
I think it's clearer to use a static attribute here since the output is pretty fixed. I would use a static method when there is some computation involved (maybe with some needed parameters).

Reply:
Fair enough :)